src/main/java/io/stargate/sgv2/jsonapi/service/bridge/config/DocumentConfig.java

```diff
@@ -46,4 +46,13 @@ public interface DocumentConfig {
   @Positive
   @WithDefault("1000")
   int maxLimit();
+
+  /**
+   * @return Defines the maximum limit of document that can be returned for a request, defaults to
+   *     <code>1000</code>.
+   */
+  @Max(100)
```

Comment:
Given that this is configuration by Admins, not user-provided, is there a reason to have Max value?

Reply:
Yes, this limit stop the admin from configuring to big numbers which will result in long running queries. Also 100 is the value @amorton gave in the discussion.